server/async_tcp.go

```diff
@@ -25,30 +32,28 @@ type AsyncServer struct {
 	maxClients             int
 	multiplexer            iomultiplexer.IOMultiplexer
 	multiplexerPollTimeout time.Duration
-	connectedClients       map[int]*core.Client
-	store                  *core.Store
+	connectedClients       map[int]*comm.Client
 	queryWatcher           *core.QueryWatcher
-	lastCronExecTime       time.Time
-	cronFrequency          time.Duration
+	shardManager           *core.ShardManager
+	ioChan                 chan *ops.StoreResponse // The server acts like a worker today, this behavior will change once IOThreads are introduced and each client gets its own worker.
 }
 
 // NewAsyncServer initializes a new AsyncServer
 func NewAsyncServer() *AsyncServer {
-	store := core.NewStore()
+	shardManager := core.NewShardManager(1)
```

Comment:
Why are we hardcoding this value to 1?
This value should come from config right?

Reply:
This will be changed once we actually support multiple shards

Reply:
this is temporary. currently server itself is acting as a IOthread -> so only one shard, 
in future when IO threads are implemented, it will be changed